README.md

```diff
@@ -12,90 +12,110 @@
+We <3 contributions big and small. The best ways to do this are:
 
-We are an open-source alternative to products such as Mixpanel, Amplitude, Heap, HotJar, Pendo or Full Story. We're designed to be more developer-friendly, with the broadest range of features like session recording, heatmaps, feature flags, and plugins.
+- Give us feedback in our [Slack community](https://posthog.com/slack)
+- Vote on features in our [roadmap](https://posthog.com/roadmap)
+- Open a PR (see our instructions on [developing PostHog locally](https://posthog.com/handbook/engineering/developing-locally))
+- Submit a [feature request](https://github.com/PostHog/posthog/issues/new?assignees=&labels=enhancement%2C+feature&template=feature_request.md) or [bug report](https://github.com/PostHog/posthog/issues/new?assignees=&labels=bug&template=bug_report.md)
 
-We play nicely with data warehouses and other services - you can _import_ event or user data by writing a plugin to create transformations, or you can _export_ data by using our existing data exports to BigQuery, Redshift, Snowflake, etc. All without losing control of your data.
+## Philosophy
 
-## Developing locally & Contributing
+Our mission is to increase the number of successful products in the world. To do that, we continue to build and improve product and data tools that help you understand user behavior without losing control of your data.
 
-See our Docs for instructions on [developing PostHog locally](https://posthog.com/docs/developing-locally).
+In our view, third-party analytics tools do not work in a world of cookie deprecation, GDPR, HIPAA, CCPA, and many other four-letter acronyms. PostHog is the alternative to sending all of your customers' personal information and usage data to third-parties.
 
-We <3 contributions big or small, check out our [guide on how to get started](https://posthog.com/docs/contributing).
+PostHog gives you every tool you need to understand user behavior, develop and test improvements, and release changes to make your product more successful.
 
-Not sure where to start? [Book a free, no-pressure pairing session](mailto:tim@posthog.com?subject=Pairing%20session&body=I'd%20like%20to%20do%20a%20pairing%20session!) with one of the team.
+PostHog operates in public as much as possible. We detail how we work and our learning on building and running a fast-growing, product-focused startup in our [handbook](https://posthog.com/handbook/getting-started/start-here).
 
-## We're hiring!
+## Open-source vs. paid
 
-Come help us make PostHog even better. We're growing fast, [and would love for you to join us](https://posthog.com/careers).
+This repo is entirely [MIT licensed](https://www.notion.so/LICENSE), except for the `ee` directory (if applicable). Need *absolutely 💯% FOSS*? Check out our [posthog-foss](https://github.com/PostHog/posthog-foss) repository, which is purged of all proprietary code and features.
```

Comment:
This notion link doesn't work and I'm not sure why we link to Notion vs our own license?

Reply:
It's what it was before, I'll rework and link to our license.